src/Components/Assets/AssetFilter.tsx

```diff
@@ -66,39 +62,43 @@ function AssetFilter(props: any) {
   const fetchFacility = useCallback(
     async (status: statusType) => {
       if (facilityId) {
-        const facilityData: any = await dispatch(
-          getPermittedFacility(facilityId)
+        const { data: facilityData } = await request(
```

Comment:
You've to use `useQuery` to perform GET requests on component mount instead of `request`